src/ProjectNeon/Assets/Features/DeckBuilder/Scripts/DeckController.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using static CharacterController;
+
+public class DeckController : MonoBehaviour
```

Comment:
Should avoid using words like `Manager` and `Controller`. Should be scoped to fixed and clear responsibilities.